File: LLM-Translations/GPT-4/codenet/C++/Go/s150958471.go
package main

import (
	"fmt"
	"bufio"
	"os"
	"strconv"
	"strings"
)

func main() {
	var train [2]int
	var bus [2]int

	reader := bufio.NewReader(os.Stdin)

	inputTrain, _ := reader.ReadString('\n')
	trainInput := strings.Fields(inputTrain)
	train[0], _ = strconv.Atoi(trainInput[0])
	train[1], _ = strconv.Atoi(trainInput[1])

	inputBus, _ := reader.ReadString('\n')
	busInput := strings.Fields(inputBus)
	bus[0], _ = strconv.Atoi(busInput[0])
	bus[1], _ = strconv.Atoi(busInput[1])

	fmt.Println(min(train[0], train[1]) + min(bus[0], bus[1]))
}

func min(a, b int) int {
	if a < b {
		return a
	}
	return b
}

// 

File: LLM-Translations/GPT-4/codenet/C++/Java/s150958471.java
import java.util.Scanner;

public class s150958471 {
    public static void main(String[] args) {
        int[] train = new int[2];
        int[] bus = new int[2];

        Scanner in = new Scanner(System.in);

        train[0] = in.nextInt();
        train[1] = in.nextInt();
        bus[0] = in.nextInt();
        bus[1] = in.nextInt();

        System.out.println(Math.min(train[0], train[1]) + Math.min(bus[0], bus[1]));

        in.close();
    }
}
// 
